tasks/perform_orca_reconcile/perform_orca_reconcile.py

```diff
@@ -278,20 +279,12 @@ def generate_mismatch_reports_sql() -> TextClause:
             files.size_in_bytes AS orca_size_in_bytes, 
             reconcile_s3_object.size_in_bytes AS s3_size_in_bytes,
             CASE 
-                WHEN (files.etag != reconcile_s3_object.etag AND files.size_in_bytes != reconcile_s3_object.size_in_bytes AND files.ingest_time !=  reconcile_s3_object.last_update) 
-                    THEN 'etag, size_in_bytes, last_update'
                 WHEN (files.etag != reconcile_s3_object.etag AND files.size_in_bytes != reconcile_s3_object.size_in_bytes) 
                     THEN 'etag, size_in_bytes'
-                WHEN (files.etag != reconcile_s3_object.etag AND files.ingest_time !=  reconcile_s3_object.last_update) 
-                    THEN 'etag, last_update'
-                WHEN (files.size_in_bytes != reconcile_s3_object.size_in_bytes AND files.ingest_time !=  reconcile_s3_object.last_update) 
-                    THEN 'size_in_bytes, last_update'
                 WHEN files.etag != reconcile_s3_object.etag 
                     THEN 'etag'
                 WHEN files.size_in_bytes != reconcile_s3_object.size_in_bytes 
                     THEN 'size_in_bytes'
-                WHEN files.ingest_time !=  reconcile_s3_object.last_update 
-                    THEN 'last_update'
```

Comment:
I know we talked about this in meeting today but could you briefly explain one more time why we are removing this?

Reply:
The last_update time in the Orca catalog is generated within the copy_to_glacier lambda, and is therefore not equivalent to the s3 last_update value.